tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -508,7 +508,7 @@ public function testHttpsRoutesWithDomains()
 
     public function testRoutesWithDomainsThroughProxy()
     {
-        Request::setTrustedProxies(['10.0.0.1'], SymfonyRequest::HEADER_X_FORWARDED_ALL);
```

Comment:
this change we could make already on 8.x?

Reply:
Not sure